### DISCUSSION TITLE
Would you be able to help me to integrate AS608 fingerprint sensor to ESP32-2432S028?

### DISCUSSION THREAD

**Discussion by AllanOricil** [rzeldent/esp32-smartdisplay#139] [Q&A]
Like I told you before, I'm new to programming in these board and I don't know much how to use these IO ports. Would you be able to teach me how I can use AS608, or R307, with my board? 

With a fingerprint sensor my mfa totp generator would be more secure. I would ask user for his fingerprint, before handing a totp code in automated sign in flows.

**Comment by AllanOricil** [comment]
@rzeldent can u help me? I bought the R307 but this A608 is cheaper, and I want to integrate it later to make my project cheaper

**Comment by rzeldent** [comment]
Hi Allan,

My time is limited, it is more a moonlighting project...
But if you have a question sur you can ask me...

**Reply by AllanOricil** [reply]
If I reach some corners which I can't figure out how to get over, I will ping you here. Do you have a discord channel or reddit community which I could also talk to people about ESP32/Arduino or any other type of microcontroller?
